packages/stores/types.ts

```diff
@@ -106,9 +112,22 @@ type TUiStore = {
     setDarkMode: (is_dark_mode_on: boolean) => boolean;
 };
 
+export type TContractStore = {
+    contract_info: ProposalOpenContract;
+    contract_update_take_profit: number & string;
+    contract_update_stop_loss: number & string;
```

Comment:
```suggestion
    contract_update_take_profit: number | string;
    contract_update_stop_loss: number | string;
```
@jim-deriv Should be `|` I guess 🤔 `number & string` will resolve to `never` type 🤔

Reply:
@farzin-deriv I used the `|` before, but didn't work, will look into it again